cql3/statements/select_statement.cc

```diff
@@ -392,6 +392,8 @@ select_statement::do_execute(query_processor& qp,
             query::is_first_page::no,
             options.get_timestamp(state));
     command->allow_limit = db::allow_per_partition_rate_limit::yes;
+    tracing::trace(state.get_trace_state(), "Executing read query (reversed {}): table schema {}, query schema {}",
+        slice.is_reversed(), _schema->version(), _query_schema->version());
```

Comment:
Most of these aren't useful for tracing. They can be useful for debugging, I think logging with trace level is more appropriate.

I'm sure we have a trace at the beginning of a query, we can add reversed there.